src/DropdownInput.js

```diff
@@ -6,9 +6,8 @@
 
 'use strict';
 
-var React = require('react/addons');
+var React = require('react');
 var ReactBootstrap = require('react-bootstrap');
-var joinClasses = require('react/lib/joinClasses');
 var cx = require('classnames');
 
 var BootstrapMixin = ReactBootstrap.BootstrapMixin;
```

Comment:
Yes, I moved to another library in another project. it's called react-widgets, and you can find the docs here : http://jquense.github.io/react-widgets/docs/#/

It has several components of different types, so depending on your use-case, it may be too bloated, or just great.